pkg/kapp/resources/mod_field_copy.go

```diff
@@ -203,3 +258,21 @@ func (t FieldCopyMod) obtainValue(obj interface{}, path Path) (interface{}, bool
 
 	return obj, true, nil
 }
+
+func (t FieldCopyMod) matchRegexWithSrcObj(regexString string, srcObj interface{}) ([]string, error) {
```

Comment:
We don't seem to be using anything from `t` (`FieldCopyMod`) here, so should we keep it as a function and reuse it in mod_field_remove.go?

Reply:
Sounds Good, Done.